controller/backend-annotations.go

```diff
@@ -258,13 +260,40 @@ func (c *HAProxyController) handleServerAnnotations(ingress *Ingress, service *S
 					continue
 				}
 				activeAnnotations = true
+			case "proxy-protocol":
+				if err := handleSendProxy(v, &server); err != nil {
```

Comment:
Resolved by e076f33, please review @Mo3m3n

Reply:
@Mo3m3n with 0764cc9 I also added tests, although adding a new dependency with `stretch/testify`

Reply:
We're almost good.
I think no need for a different commit [Resolving discussion_r438284031](https://github.com/haproxytech/kubernetes-ingress/pull/204/commits/ee339b4a4084bb19c7c245349c6155cd1132170b), if you rebase the first one to update the changes accordingly it will be perfect.
Also while rebasing you can also increment this (tiny detail) [number](https://github.com/haproxytech/kubernetes-ingress/blob/aa3cb669733aed3729944b067b16fc34ad5c1647/controller/backend-annotations.go#L210)